source/utils/get-charset.ts

```diff
@@ -28,7 +28,7 @@ export default function getCharset(content: Buffer, headers?: Headers) {
 
 		charset = parseContentType(
 			$('meta[charset]').attr('charset') // HTML5
-			|| $('meta[http-equiv=Content-Type][content]').attr('content') // HTML4
+			|| parseContentType($('meta[http-equiv="Content-Type"][content]').attr('content')) // HTML4
```

Comment:
@Ndpnt I do not understand how this change will prevent `X-UA-Compatible` from being taken into account.

`meta[http-equiv="Content-Type"][content]` is already
 getting only `<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">` and not `<meta http-equiv="X-UA-Compatible" content="IE=edge">`


![Screen Shot 2022-09-20 at 12 38 20](https://user-images.githubusercontent.com/4191809/191210789-6fe3a553-35ef-4024-99f4-581a0b1af971.png)

Reply:
Ok, I did the job that was already done… But not published.
See https://github.com/node-fetch/fetch-charset-detection/issues/249